test/models/test_albef.py

```diff
@@ -4,113 +4,220 @@
-        set_rng_seed(0)
-        self.albef.image_queue = torch.randn(2, 4)
-        self.albef.text_queue = torch.randn(2, 4)
-        image_feat = torch.randn(2, 2)
-        text_feat = torch.randn(2, 2)
-        image_feat_m = torch.randn(2, 2)
-        text_feat_m = torch.randn(2, 2)
-        output = self.albef._similarity(
-            image_feat, text_feat, image_feat_m, text_feat_m
-        )
-        expected_sim_i2t = Tensor(
-            [
-                [-3.660729, -11.191917, 1.252719, 9.129601, -0.882724, -0.818697],
-                [15.755546, 21.687363, 27.634123, -18.587852, 30.696188, -4.964930],
-            ]
-        )
-        expected_sim_t2i = Tensor(
-            [
-                [27.311251, -23.288742, -14.411922, -30.653456, -3.136197, 20.444725],
-                [16.565296, -26.082125, 8.684321, -19.420963, -24.787359, 16.908016],
-            ]
-        )
-        expected_sim_i2t_m = Tensor(
-            [
-                [-13.028821, -22.274969, -17.438065, 18.764980, -20.974815, 2.714235],
-                [8.504787, 5.272466, 22.941040, -5.002890, 23.104834, -4.742508],
-            ]
-        )
-        expected_sim_t2i_m = Tensor(
-            [
-                [-13.028821, 8.504787, 10.671893, 14.442708, -3.490067, -8.771053],
-                [-22.274969, 5.272466, 31.752522, 24.050060, -23.705740, -11.501695],
-            ]
-        )
-        assert_expected(output.sim_i2t, expected_sim_i2t, rtol=0, atol=1e-4)
-        assert_expected(output.sim_t2i, expected_sim_t2i, rtol=0, atol=1e-4)
-        assert_expected(output.sim_i2t_m, expected_sim_i2t_m, rtol=0, atol=1e-4)
-        assert_expected(output.sim_t2i_m, expected_sim_t2i_m, rtol=0, atol=1e-4)
-
-    def test_neg_embeddings(self):
-        set_rng_seed(0)
-        image_embeds = torch.randn(2, 1, 3)
-        text_embeds = torch.randn(2, 1, 3)
-        text_atts = torch.randn(2, 1)
-        similarity = ALBEFSimilarity(
-            sim_i2t=torch.randn(2, 5),
-            sim_t2i=torch.randn(2, 5),
-            sim_i2t_m=torch.randn(2, 5),
-            sim_t2i_m=torch.randn(2, 5),
-        )
-        image_embeds_neg, text_embeds_neg, text_atts_neg = self.albef._neg_embeddings(
-            image_embeds, text_embeds, text_atts, similarity
-        )
-        expected_image_embeds_neg = Tensor(
-            [[0.568431, -1.084522, -1.398595], [1.540996, -0.293429, -2.178789]]
-        ).unsqueeze(1)
-        expected_text_embeds_neg = Tensor(
-            [[-0.403344, -0.596635, 0.182036], [0.403347, 0.838026, -0.719258]]
-        ).unsqueeze(1)
-        expected_text_atts_neg = Tensor([1.100604, -0.856675]).unsqueeze(1)
-        assert_expected(image_embeds_neg, expected_image_embeds_neg, rtol=0, atol=1e-4)
-        assert_expected(text_embeds_neg, expected_text_embeds_neg, rtol=0, atol=1e-4)
-        assert_expected(text_atts_neg, expected_text_atts_neg, rtol=0, atol=1e-4)
+
+@pytest.fixture(autouse=True)
+def albef_model_output(albef_model):
```

Comment:
Similar comment for autouse here, and on the usage of `albef_model_output` and `dummy_albef_model` below